backfill_mismatched_deposits.sql

```diff
@@ -0,0 +1,64 @@
+-- soft-delete events for overcounted deposits
+UPDATE events SET deleted_at = current_timestamp
```

Comment:
Do we also need to set points to 0 here? It looks like we do that [when 'deleting' deposits](https://github.com/iron-fish/ironfish-api/blob/61b1cde0b82e412ab691ddc90c5e9e2ff2c27cac/src/events/events.service.ts#L619)

Reply:
Will update!